tasklist/common/src/main/java/io/camunda/tasklist/os/OpenSearchConnector.java

```diff
@@ -105,25 +105,22 @@ public OpenSearchClient openSearchClient() {
   }
 
   @Bean
-  public RestClient opensearchRestClient() {
-    final var httpHost = getHttpHost(tasklistProperties.getOpenSearch());
-    return RestClient.builder(httpHost)
-        .setHttpClientConfigCallback(
-            b -> configureApacheHttpClient(b, tasklistProperties.getOpenSearch()))
-        .build();
+  public OpenSearchClient tasklistZeebeOsClient() {
+    System.setProperty("es.set.netty.runtime.available.processors", "false");
+    return createOsClient(tasklistProperties.getZeebeOpenSearch());
   }
 
   @Bean
-  public RestClient opensearchZeebeRestClient() {
-    final var httpHost = getHttpHost(tasklistProperties.getZeebeOpenSearch());
+  public RestClient tasklistOpensearchRestClient() {
```

Comment:
If file comment is irrelevant, this should be _tasklistOpenSearchRestClient_ (camelCase OpenSearch). Along with it's qualifiers

Reply:
I used the pattern from the other comment